src/bicgstab.cpp

```diff
@@ -108,7 +108,7 @@ static void xpay(int n, realnum *x, double a, const realnum *y) {
 typedef realnum *prealnum; // grr, ISO C++ forbids new (double*)[...]
 
 /* BiCGSTAB(L) algorithm for the n-by-n problem Ax = b */
-int bicgstabL(const int L, const int n, realnum *x,
+meep::integer bicgstabL(const meep::integer L, const meep::integer n, realnum *x,
```

Comment:
The return value here is just an error code; why can't it be an `int`?  And `L` is the size of a subspace that will always be `< 1000` in practice; why did this need to change?